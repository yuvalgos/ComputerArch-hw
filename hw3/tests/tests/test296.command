./dflow_calc tests/test296.dat tests/test296.in d106 d173 d194 p0 p7 p177 p147 p180 p161 d48 d81 p140 d105 d14 d151 d40 p73 d163 d127 p0 d124 p132 p44 d78 p122 d181 p15 d116 p81 p188 d146 d101 d11 d128 d117 d106 d50 d103 p22 d44 p34 d83 d38 p53 p62 p143 d48 p58 p42 p7 p68 p66 d115 p121 p113 d33 d179 p45 d89 p89 d190 d14 p127 d165 d140 p112 d20 d123 d166 d80 d116 p5 d122 p155 p153 d53 d138 p194 d50 d140 d158 d145 p17 d196 d171 p170 p81 p113 d138 d123 d109 d78 d196 d191 d189 p16 p120 p81 d183 p132 p121 p104 d179 d93 d86 d18 p85 d68 d186 d78 p116 p167 d110 d174