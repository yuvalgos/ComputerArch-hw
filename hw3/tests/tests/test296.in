6 16 4 21
1 5 10 31
1 10 3 31
0 24 13 31
5 4 1 29
4 27 10 5
5 21 13 30
5 20 0 19
4 17 10 20
4 5 21 13
2 4 5 20
2 24 31 26
5 22 20 28
2 14 26 17
7 20 24 4
6 21 3 7
4 27 3 20
0 20 15 25
7 19 10 13
3 2 7 27
5 30 15 30
1 5 30 3
7 25 18 20
8 20 3 2
3 16 23 24
1 24 9 6
2 18 10 28
1 28 1 3
7 13 14 1
6 9 31 29
6 17 24 2
5 22 7 29
1 29 21 30
5 31 30 6
1 19 19 22
6 3 29 21
1 2 30 3
2 24 4 5
5 8 8 24
8 1 17 20
3 8 9 12
0 23 30 25
2 30 15 17
8 2 8 1
3 8 21 15
7 17 19 14
1 24 24 2
5 19 28 17
4 7 29 18
2 18 6 19
2 23 21 21
7 16 31 30
5 19 4 20
8 1 18 4
6 20 28 26
8 15 24 14
6 1 5 26
0 12 4 15
1 10 18 14
4 1 23 25
7 9 19 29
1 29 2 23
1 26 25 17
7 12 19 27
6 24 19 20
2 17 5 22
7 13 4 3
5 11 2 9
0 21 13 21
6 25 9 7
6 1 15 10
1 12 1 5
8 13 20 9
5 9 13 0
7 5 25 9
0 10 27 7
2 24 24 3
4 24 10 12
8 23 1 16
5 11 8 21
0 2 4 30
1 31 27 15
6 8 22 27
0 22 22 19
8 7 29 7
7 3 3 26
4 14 6 21
7 6 10 17
1 3 19 17
8 1 25 9
1 28 5 11
4 2 19 26
2 22 3 28
1 29 25 24
1 17 4 27
3 10 22 3
5 7 30 10
6 8 8 28
8 11 23 4
3 4 13 9
7 23 24 14
0 8 29 28
1 8 31 29
6 11 8 4
0 26 22 26
3 3 13 0
3 19 20 5
4 3 27 13
1 3 28 4
1 20 2 17
2 17 14 20
0 18 17 6
4 6 8 12
1 3 10 23
5 27 10 9
7 2 11 15
1 12 5 0
6 7 27 19
1 5 28 25
4 15 16 21
8 28 3 8
2 19 11 15
4 11 25 21
5 8 28 24
0 5 14 6
1 5 22 10
1 0 24 8
1 13 24 6
3 4 16 6
5 11 17 26
3 20 0 27
0 21 12 24
7 25 21 28
8 27 23 10
2 23 0 28
0 11 8 9
3 29 7 6
5 28 26 0
1 23 15 17
5 14 15 16
4 26 29 5
5 23 14 15
1 12 31 21
7 18 20 6
4 17 23 4
6 14 9 16
6 23 28 17
5 16 8 25
7 4 14 6
0 0 12 6
8 20 20 13
7 29 8 5
5 23 21 0
8 11 29 0
0 2 3 20
2 7 2 4
8 19 28 18
7 8 14 30
7 17 2 24
1 5 20 19
7 8 13 5
5 2 30 12
2 29 16 18
8 1 18 20
6 24 13 5
7 11 11 0
8 2 10 29
0 23 19 22
2 15 3 10
6 17 3 18
8 28 11 5
7 20 26 8
7 16 28 3
5 4 20 27
2 26 26 20
2 0 8 23
6 27 18 0
5 8 11 27
4 10 23 3
5 5 14 29
2 27 29 0
6 6 12 28
0 29 12 7
0 27 17 19
7 14 21 4
7 16 5 8
3 31 1 15
1 7 4 10
1 28 31 31
1 29 18 27
5 3 18 25
4 25 1 31
8 5 4 15
3 2 10 22
3 4 27 6
5 3 17 23
6 28 19 4
5 29 29 24
1 11 25 10
1 20 14 4
