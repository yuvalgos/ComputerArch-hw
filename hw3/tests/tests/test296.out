Reading the opcodes latency info from tests/test296.dat ... Got latency for 9 opcodes
Reading the program file tests/test296.in ... Found 200 instructions
getProgDepth()==379
getInstDeps(106)=={54,74}
getInstDeps(173)=={171,133}
getInstDeps(194)=={183,181}
getDepDepth(0)==0
getDepDepth(7)==0
getDepDepth(177)==346
getDepDepth(147)==256
getDepDepth(180)==329
getDepDepth(161)==197
getInstDeps(48)=={32,26}
getInstDeps(81)=={16,55}
getDepDepth(140)==228
getInstDeps(105)=={72,-1}
getInstDeps(14)=={11,10}
getInstDeps(151)=={123,125}
getInstDeps(40)=={29,-1}
getDepDepth(73)==144
getInstDeps(163)=={143,150}
getInstDeps(127)=={77,112}
getDepDepth(0)==0
getInstDeps(124)=={86,112}
getDepDepth(132)==237
getDepDepth(44)==17
getInstDeps(78)=={70,51}
getDepDepth(122)==138
getInstDeps(181)=={142,170}
getDepDepth(15)==0
getInstDeps(116)=={74,-1}
getDepDepth(81)==161
getDepDepth(188)==313
getInstDeps(146)=={137,144}
getInstDeps(101)=={93,90}
getInstDeps(11)=={-1,-1}
getInstDeps(128)=={51,112}
getInstDeps(117)=={114,106}
getInstDeps(106)=={54,74}
getInstDeps(50)=={15,15}
getInstDeps(103)=={102,99}
getDepDepth(22)==2
getInstDeps(44)=={15,-1}
getDepDepth(34)==22
getInstDeps(83)=={31,52}
getInstDeps(38)=={-1,37}
getDepDepth(53)==66
getDepDepth(62)==48
getDepDepth(143)==260
getInstDeps(48)=={32,26}
getDepDepth(58)==66
getDepDepth(42)==83
getDepDepth(7)==0
getDepDepth(68)==56
getDepDepth(66)==37
getInstDeps(115)=={103,55}
getDepDepth(121)==215
getDepDepth(113)==211
getInstDeps(33)=={20,-1}
getInstDeps(179)=={145,162}
getDepDepth(45)==29
getInstDeps(89)=={69,73}
getDepDepth(89)==157
getInstDeps(190)=={143,132}
getInstDeps(14)=={11,10}
getDepDepth(127)==209
getInstDeps(165)=={153,149}
getInstDeps(140)=={136,125}
getDepDepth(112)==198
getInstDeps(20)=={-1,-1}
getInstDeps(123)=={120,77}
getInstDeps(166)=={95,162}
getInstDeps(80)=={10,42}
getInstDeps(116)=={74,-1}
getDepDepth(5)==3
getInstDeps(122)=={69,68}
getDepDepth(155)==285
getDepDepth(153)==269
getInstDeps(53)=={49,10}
getInstDeps(138)=={119,110}
getDepDepth(194)==351
getInstDeps(50)=={15,15}
getInstDeps(140)=={136,125}
getInstDeps(158)=={154,77}
getInstDeps(145)=={73,51}
getDepDepth(17)==0
getInstDeps(196)=={156,194}
getInstDeps(171)=={140,160}
getDepDepth(170)==311
getDepDepth(81)==161
getDepDepth(113)==211
getInstDeps(138)=={119,110}
getInstDeps(123)=={120,77}
getInstDeps(109)=={91,94}
getInstDeps(78)=={70,51}
getInstDeps(196)=={156,194}
getInstDeps(191)=={163,186}
getInstDeps(189)=={143,183}
getDepDepth(16)==48
getDepDepth(120)==214
getDepDepth(81)==161
getInstDeps(183)=={169,156}
getDepDepth(132)==237
getDepDepth(121)==215
getDepDepth(104)==184
getInstDeps(179)=={145,162}
getInstDeps(93)=={69,77}
getInstDeps(86)=={-1,68}
getInstDeps(18)=={2,-1}
getDepDepth(85)==51
getInstDeps(68)=={66,15}
getInstDeps(186)=={163,168}
getInstDeps(78)=={70,51}
getDepDepth(116)==176
getDepDepth(167)==302
getInstDeps(110)=={86,109}
getInstDeps(174)=={140,171}
